metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/monitor/ConsumerInvocationMonitor.java

```diff
@@ -17,11 +17,12 @@
 
 package org.apache.servicecomb.metrics.core.monitor;
 
+import java.util.HashMap;
+import java.util.Map;
 
-import org.apache.servicecomb.metrics.common.ConsumerInvocationMetric;
-import org.apache.servicecomb.metrics.common.MetricsConst;
+import org.apache.servicecomb.foundation.metrics.MetricsConst;
 
-public class ConsumerInvocationMonitor extends InvocationMonitor {
+public class ConsumerInvocationMonitor {
```

Comment:
What we should do if we want to change the ConsumerInvocationMonitor into ProducerInvocationMonitor？We don't need to write lots Classes here, please use abstraction to reduce the number of the class.

Reply:
I had use DefaultMonitorRegistry as monitor container replace it;
In order to update monitor,I had add a group of basic wrapped monitor extends from AbstractMonitor;
Measure metrics will direct from DefaultMonitorRegistry.getInstance().getRegisteredMonitors() then getValue()